### PR TITLE
Show parent org users in sub org role

### PR DESCRIPTION
**Pull request by ravinduramesh**
Screenshot 2023-04-26 at 15 57 07

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/roles/components/wizard/role-user-assign.tsx [hunk 1]
```diff
@@ -579,30 +579,77 @@ export const AddRoleUsers: FunctionComponent<AddRoleUserProps> = (props: AddRole
...
+                                                                                    floated="left"
+                                                                                />
+                                                                            </Table.Cell>
+                                                                            <Table.Cell>
+                                                                                { user.display }
+                                                                            </Table.Cell>
+                                                                            <Table.Cell>
+                                                                                {
+                                                                                    user.orgName
+                                                                                    === "Super" ? "Asgardeo" :
```
Comment (NipuniBhagya) [inline apps/console/src/features/roles/components/wizard/role-user-assign.tsx:613]: Let's not use hardcoded values here. Better to use a config instead.

**Review by NipuniBhagya — CHANGES_REQUESTED** [review]

File: apps/console/src/features/roles/components/wizard/role-user-assign.tsx (hunk 1 above)
Comment (ravinduramesh) [reply apps/console/src/features/roles/components/wizard/role-user-assign.tsx]: Resolved with https://github.com/wso2/identity-apps/pull/3887/commits/c2eba0e23012ea071441b09a2445e2fd1b241868

File: apps/console/src/features/organizations/components/edit-organization-role/edit-organization-groups.tsx [hunk 2]
```diff
@@ -287,14 +291,14 @@ export const RoleGroupsList: FunctionComponent<RoleGroupsPropsInterface> = (
         setAddNewRoleModalView(false);
     };
 
-    const handleUnselectedListSearch = (e, { value }) => {
-        let isMatch = false;
-        const filteredGroupList = [];
+    const handleUnselectedListSearch = (_e: any, { value }: any) => {
```
Comment (thivi) [inline apps/console/src/features/organizations/components/edit-organization-role/edit-organization-groups.tsx:294]: Let's avoid using any

File: apps/console/src/features/organizations/components/edit-organization-role/edit-organization-groups.tsx [hunk 3]
```diff
@@ -306,14 +310,14 @@ export const RoleGroupsList: FunctionComponent<RoleGroupsPropsInterface> = (
         }
     };
 
-    const handleSelectedListSearch = (e, { value }) => {
-        let isMatch = false;
-        const filteredGroupList = [];
+    const handleSelectedListSearch = (_e: any, { value }: any) => {
```
Comment (thivi) [inline apps/console/src/features/organizations/components/edit-organization-role/edit-organization-groups.tsx:313]: Let's avoid using any

File: apps/console/src/features/roles/components/edit-role/edit-role-groups.tsx [hunk 4]
```diff
@@ -279,14 +282,14 @@ export const RoleGroupsList: FunctionComponent<RoleGroupsPropsInterface> = (
         setAddNewRoleModalView(false);
     };
 
-    const handleUnselectedListSearch = (e, { value }) => {
-        let isMatch = false;
-        const filteredGroupList = [];
+    const handleUnselectedListSearch = (_e: any, { value }: any) => {
```
Comment (thivi) [inline apps/console/src/features/roles/components/edit-role/edit-role-groups.tsx:285]: Let's avoid any

File: apps/console/src/features/roles/components/edit-role/edit-role-groups.tsx [hunk 5]
```diff
@@ -298,14 +301,14 @@ export const RoleGroupsList: FunctionComponent<RoleGroupsPropsInterface> = (
         }
     };
 
-    const handleSelectedListSearch = (e, { value }) => {
-        let isMatch = false;
-        const filteredGroupList = [];
+    const handleSelectedListSearch = (_e: any, { value }: any) => {
```
Comment (thivi) [inline apps/console/src/features/roles/components/edit-role/edit-role-groups.tsx:304]: Let's avoid any

File: apps/console/src/features/roles/components/edit-role/edit-role-groups.tsx [hunk 6]
```diff
@@ -336,25 +339,25 @@ export const RoleGroupsList: FunctionComponent<RoleGroupsPropsInterface> = (
...
             method: "PATCH",
             path: "/Roles/" + role.id
         };
 
-        const removeOperations = [];
-        const addOperations = [];
-        let removedIds = [];
-        const addedIds = [];
+        const removeOperations: any = [];
+        const addOperations: any = [];
```
Comment (thivi) [inline apps/console/src/features/roles/components/edit-role/edit-role-groups.tsx:351]: Let's avoid any

File: apps/console/src/features/roles/components/wizard/role-user-assign.tsx [hunk 7]
```diff
@@ -579,30 +580,78 @@ export const AddRoleUsers: FunctionComponent<AddRoleUserProps> = (props: AddRole
...
+                                                                                        size="mini"
+                                                                                        floated="left"
+                                                                                    />
+                                                                                </Table.Cell>
+                                                                                <Table.Cell>
+                                                                                    { selectedUser.userName }
+                                                                                </Table.Cell>
+                                                                                <Table.Cell>
+                                                                                    Not Resolved
+                                                                                </Table.Cell>
```
[the remark this answers is not in the document]
Comment (ravinduramesh) [reply apps/console/src/features/roles/components/wizard/role-user-assign.tsx:649]: Resolved in https://github.com/wso2/identity-apps/pull/3887/commits/25602fccbd2e8e669bb648d3dedb21ee447ff531

File: apps/console/src/features/roles/components/wizard/role-user-assign.tsx [hunk 8]
```diff
@@ -579,30 +580,77 @@ export const AddRoleUsers: FunctionComponent<AddRoleUserProps> = (props: AddRole
...
+                                                                            </Table.Cell>
+                                                                            <Table.Cell>
+                                                                                { user.display }
+                                                                            </Table.Cell>
+                                                                            <Table.Cell>
+                                                                                {
+                                                                                    user.orgId
+                                                                                    === OrganizationManagementConstants
+                                                                                        .ROOT_ORGANIZATION_ID ? 
+                                                                                        "Asgardeo" : user.orgName
```
Comment (NipuniBhagya) [inline apps/console/src/features/roles/components/wizard/role-user-assign.tsx:616]: Let's remove this hardcoded values.

**Review by NipuniBhagya — CHANGES_REQUESTED** [review]

File: apps/console/src/features/roles/components/wizard/role-user-assign.tsx (hunk 8 above)
Comment (ravinduramesh) [reply apps/console/src/features/roles/components/wizard/role-user-assign.tsx]: Resolved with https://github.com/wso2/identity-apps/pull/3887/commits/50e5e9f56cda09b88200af2487ac61e9ebf43808

**Review by thivi — DISMISSED** [review]

File: apps/console/src/features/roles/components/wizard/role-user-assign.tsx (hunk 8 above)
Comment (ravinduramesh) [reply apps/console/src/features/roles/components/wizard/role-user-assign.tsx]: Resolved with https://github.com/wso2/identity-apps/pull/3887/commits/969313ac6c03dbca237c7f22c785b9ad2d35bd99

**Review by NipuniBhagya — APPROVED** [review]

**Pull request merged**